client_test.go

```diff
@@ -349,7 +349,8 @@ func TestClient_Create(t *testing.T) {
 			name:       "Create with no body",
 			client:     defaultClient,
 			collection: migrations.PostsPublic,
-			wantErr:    true,
+			wantErr:    false,
+			wantID:     true,
```

Comment:
This works because there are no required fields. I'm not sure if it worked before or not.